src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/mappers/SupplementaryEvidenceMapperTest.java

```diff
@@ -25,14 +25,18 @@ void maps_all_fields_correctly() {
         // given
         List<Document> existingDocs =
             asList(
-                new Document("a.pdf", "aaa", "type_a", "subtype_a", now().plusSeconds(1), "http://localhost/a.pdf"),
-                new Document("b.pdf", "bbb", "type_b", "subtype_b", now().plusSeconds(2), "http://localhost/b.pdf")
+                new Document(
+                    "a.pdf", "aaa", "type_a", "subtype_a", now().plusSeconds(1), "http://localhost/a.pdf", "uuida"
```

Comment:
I don’t mind using a dummy value here but atleast format should be correct. Docuuid is part of docurl looking at this it looks like they can be completely different

Reply:
I'll update the tests with the correct doc url format

Reply:
Done. Updated all unit tests data to include the document uuid in the document url